internal/graph/reachableresources.go

```diff
@@ -549,7 +549,6 @@ func (crr *CursoredReachableResources) redispatchOrReport(
 				Metadata: &v1.ResolverMeta{
 					AtRevision:     parentRequest.Revision.String(),
 					DepthRemaining: parentRequest.Metadata.DepthRemaining - 1,
-					RequestId:      parentRequest.Metadata.RequestId,
```

Comment:
In the places where we're just removing request ID, why didn't we also add the bloom filter?
Isn't it possible that reachable resources could also fall into a loop?

Reply:
Singleflight dispatcher is only used on check and expand, so I think my thought was to only add this where needed.